### DISCUSSION TITLE
❓ PyTorch 1.5.1 load() missing 1 required positional argument: 'github'

### DISCUSSION THREAD

**Discussion by garymmi** [snakers4/silero-vad#31] [Q&A] [answered]
## ❓ Questions and Help

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.

i run the vad example, error as below
Traceback (most recent call last):
  File "vad.py", line 7, in <module>
    force_reload=True)
TypeError: load() missing 1 required positional argument: 'github'

i use torch 1.5.1

thanks

**Comment by snakers4** [comment]
> i use torch 1.5.1

Please use 1.7+ 
I need to add this into docs

**Comment by snakers4** [comment, marked as answer]
Added this section - https://github.com/snakers4/silero-vad#pytorch

Please note that if you just use PyTorch 1.7+ all other dependencies except for PyTorch itself are mostly superficial and are just used for IO only (see the utils file).

PyTorch 1.7 introduced breaking changes in `torch.hub` and since PyTorch 1.7 in a master version in most package managers, we use it by default

**Comment by snakers4** [comment]
Similar to this - https://github.com/snakers4/silero-models/issues/26

**Comment by Preet-Garach** [comment]
I am using 2.0.1, still facing this issue on aws sagemaker
